.travis.yml

```diff
@@ -23,7 +24,32 @@ matrix:
     - rust: nightly
       env: FEATURES=""
   allow_failures:
-    - rust: nightly
+  - rust: nightly
 
 script:
   - cargo test --no-default-features --features "$FEATURES"
+
+before_deploy:
+- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    make dmg;
+    export DEPLOY_FILE="$PWD/target/release/osx/Alacritty.dmg";
+  fi
+- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+    cargo build --release;
+    cargo install cargo-deb;
+    export DEPLOY_FILE=$( cargo deb --no-build );
+  fi
+- echo "DEPLOYING FILE $DEPLOY_FILE";
+
+deploy:
+  provider: releases
+  api_key:
+    secure: o2Q39A2JAKeMEYD0dFJTpYJMpAsHYcUa4DIXApqn/ygXQzcfPpnLelCWb70zOlICIk3ICETlzeLfF7s/q802jy5aEFLbbmIp+rT0SWlRiShuQqLPkwlK7Cqh2nUAVPoozYn1y9ha8kozOL3xgDxI9RGHMC+7ZV9duVLWUqz5Ah7qBoOx9QemDMrdPJ7zi8vFvJp4oW3EV7wQRRfEmFkqOAxbAWhLeTNGVR9ECjUZIgXWRz1kjqVz6dJrLauH/D068RSV/7JM1dlzrMkBsmRe8xVOTnAiHIoGMB4UaUTLz8hJNH+KUmXfprfBKuFEQ7jDnYq+6/y200VYuRfRsU2EOqYjzHXmWD6E3vkHhVdrUUfFmG9tfAzIXFZxs0LVIszWgqqMypmuLNxgYcfpMmo+YeHGWX59HOVwXmvY38ix1++zLh44kpRuX0mYFuPdH6gRdeKPr7jyueODKRwsegh1d8rVl7T0HQpBmDcWHBidOf8Aw8NTrsEoRQFlS/eVZyGnZet/nf6Otenp/5MIBA3BHO9R9QaCo0FT3Yi7FL/kkoEZs+lN5mKJulZ6hpmG760h+mYBqw7buLKcCR3qBiuXOpgfzNWgOoAGb3G7eUAhCf1zHTEcdL6U/SEA2ofpaPZGqrcuZ7aTgnJ6lWU4YWI8X+hAouv6IOk/UoxiONIUwHE=
+  skip_cleanup: true
+  overwrite: true
+  file_glob: true
+  file: $DEPLOY_FILE
+  on:
+    rust: stable
+    all_branches: true
+    condition: '$TRAVIS_TAG != "" || $TRAVIS_EVENT_TYPE = "cron"'
```

Comment:
Looking right at the top of the travis documentation (https://docs.travis-ci.com/user/deployment/releases/), it should be possible to just use `on: tags: true`.

Reply:
Using `tags: true` wouldn't work for building nightlies. I can definitely set it, but then any travis-based cron job wouldn't trigger the deploy process. I don't mind either way though.

Reply:
What do you mean by `travis-based cron job`? We'd only want to release `.deb` files for stable releases that are manually tagged through `git`. I don't think providing nightly `.deb`s should be a focus for now, for nightly compiling yourself seems like the way to go.

If we would release nightlies instead of only commits tagged on github we'd probably run into issues with a ton of people running nightly instead of the stable version. It seems like a smart choice to try and get most people to use the tagged releases.

Reply:
Having a PPA with nightlies would be great though. It takes quite a while to rebuild Alacritty if you update the rust compiler and don't have the fastest machine in the world.

Reply:
Yeah that's a good argument @NickeZ, but with releases on github I'd stick with stable releases first. I'm not quite sure how it works either, but I think with github the latest version (in that case nightly) would always be the first element and thus reduce the visibility of stable releases.

If there is a convinient way to do this, it would be fine though. However it should still be disabled on nightly builds, maybe if we want to push nightlies it should be a separate job that just gets automatically triggered every night to release them. But I'd say something like that is so far out that I wouldn't worry too much about it yet.